tests/test_graph/test_algorithms.py

```diff
@@ -0,0 +1,188 @@
+"""Tests for graph algorithms."""
+
+from typing import Literal, TypeAlias
+
+import jax
+import numpy as np
+import pytest
+
+import causalprog
+from causalprog.distribution.normal import NormalFamily
+from causalprog.graph import DistributionNode, Graph, ParameterNode
+
+NormalGraphNodeNames: TypeAlias = Literal["mean", "cov", "outcome"]
+NormalGraphNodes: TypeAlias = dict[
+    NormalGraphNodeNames, DistributionNode | ParameterNode
+]
```

Comment:
Are these used anywhere in this file?

```suggestion
```

They could be used to typehint the fixtures that the tests used ofc, but I think that's overkill in this file. Especially since you've already typehinted the fixtures themselves.